synapse/lib/layer.py

```diff
@@ -1406,11 +1406,17 @@ async def setLayerInfo(self, name, valu):
         '''
         Set a mutable layer property.
         '''
-        if name not in ('name',):
+        if name not in ('name', 'logedits'):
             mesg = f'{name} is not a valid layer info key'
             raise s_exc.BadOptValu(mesg=mesg)
+
+        if name == 'logedits':
+            valu = bool(valu)
+            self.logedits = valu
```

Comment:
don't we need to persist it?

Reply:
yep, that's what the line below already does.